Shared/Extensions/DispatcherQueueExtensions.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Windows.Foundation.Metadata;
+using Windows.System;
+
+namespace MessageDialogService.Extensions
```

Comment:
The namespace is not aligned with other extension methods. We are currently using the namespace of the extended type instead of a custom namespace.